include/deal.II/base/quadrature_point_data.h

```diff
@@ -438,10 +438,10 @@ namespace parallel
       FullMatrix<double> matrix_quadrature;
 
       /**
-       * The offset that the parallel::distributed::Triangulation has assigned
-       * to this object starting at which we are allowed to read.
+       * The handle that the parallel::distributed::Triangulation has assigned
+       * to this object while registering the pack_callback function.
        */
-      unsigned int offset;
+      unsigned int handle;
```

Comment:
Good job finding these places. This sort of patch where you just rename things without changing the functionality can also easily go into separate patches -- they're typically quite easy to approve.